### PR TITLE
Add 7.2 and static analysis checks back to travis

### PR DESCRIPTION
**Pull request by markstory**
We can remove them again once we are able to turn github checks back on.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -50,9 +57,15 @@ script:
       if [[ $TRAVIS_PHP_VERSION == '7.4snapshot' ]]; then
         # This is necessary to skip memory usage tests
         CODECOVERAGE=1 vendor/bin/phpunit --verbose --coverage-clover=clover.xml
-      else
+      elif [[ $DEFAULT == '1' ]]; then
         vendor/bin/phpunit
       fi
+  - |
+      if [[ $CHECKS == 1 ]]; then
+        composer phpstan-setup
```
Comment (ADmad) [inline .travis.yml]: Not running pslam checks?

**Comment by ADmad** [comment]
Hmm. A separate job for checks never showed up on travis.

File: .travis.yml [hunk 2]
```diff
@@ -14,13 +14,20 @@ cache:
...
 
 env:
+  global:
+    - DEFAULT=1
+    - CHECKS=0
   matrix:
     - DB=mysql DB_DSN='mysql://root@127.0.0.1/cakephp_test?init[]=SET sql_mode = "STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION"'
     - DB=pgsql DB_DSN='postgres://postgres@127.0.0.1/cakephp_test'
     - DB=sqlite DB_DSN='sqlite:///:memory:'
+  include:
```
Comment (othercorey) [inline .travis.yml]: This needs to go under `matrix:` section not `env:` section

**Comment by othercorey** [comment]
```
-'Tue, 19 Nov 2019 04:02:01 GMT'
+'Tue, 19 Nov 2019 04:02:02 GMT'
/home/travis/build/cakephp/cakephp/tests/TestCase/Http/ResponseTest.php:480
```

I'm guessing we have an invalid test setup.

**Pull request merged**